README.md

```diff
@@ -1,20 +1,69 @@
-### Install NPM dependencies
+# BMC-UI
 
-```js
-npm install | npm i
-```
+BMC-UI is a web-based user interface for managing and configuring the BMC of a Turing Pi cluster.
 
-### Run Server & Watcher
+## Technologies Used
 
-```bash
-npm run dev | gulp
-```
+- [React](https://reactjs.org/) - JavaScript library for building user interfaces
+- [TanStack Router](https://tanstack.com/router) - Routing library for React applications
+  - Handles route management and route protection
+- [TanStack Query](https://tanstack.com/query) - Data fetching and caching library for React
+- [Vite](https://vitejs.dev/) - Fast build tool and development server
+- [TypeScript](https://www.typescriptlang.org/) - Typed superset of JavaScript
 
-### Build Project
+## Getting Started
 
-```bash
-npm run build | gulp build
-```
+1. Set up `bmcd-api-mock` for development:
+   - Clone the [bmcd-api-mock](https://github.com/barrenechea/bmcd-api-mock) repository.
+   - Follow the instructions in the `bmcd-api-mock` repository to set up and run the mock server.
```

Comment:
I'm not sure if I should leave this in. It helped me develop the new app, but it's not a "first-party solution" from Turing. Open to suggestions!

I guess an option would be for the dev to target its own cluster. I'm not sure if CORS restrictions apply there.

Reply:
yea, I did something similar with enabling CORS and, therefore, running the web page on my local machine, passing the API calls to the Turing board.

It could be useful, not sure if you are willing to donate it. But it would at least be worthy of a comment about its existence in the docs

Reply:
I'd be happy to donate the code, although I'd like to polish it a bit, like making it more modular and more "to-the-spec", specifically on error handling 😄